Flick/Cells/SuggestionTableViewCell.swift

```diff
@@ -160,11 +159,11 @@ class SuggestionTableViewCell: UITableViewCell {
             make.centerY.equalTo(movieIconImageView)
         }
 
-        tagsLabel.snp.makeConstraints { make in
-           make.leading.equalTo(spacerView.snp.trailing).offset(6)
-           make.centerY.equalTo(movieIconImageView)
-           make.trailing.equalTo(containerView).inset(padding)
-        }
+//        tagsLabel.snp.makeConstraints { make in
+//           make.leading.equalTo(spacerView.snp.trailing).offset(6)
+//           make.centerY.equalTo(movieIconImageView)
+//           make.trailing.equalTo(containerView).inset(padding)
+//        }
```

Comment:
commented out for now since we dont get tags in the response and there's not really enough space for it